cgroups/fs/apply_raw.go

```diff
@@ -137,7 +138,10 @@ func (m *Manager) GetPaths() map[string]string {
 
 func (m *Manager) GetStats() (*cgroups.Stats, error) {
 	stats := cgroups.NewStats()
-	for name, path := range m.Paths {
+	m.Lock()
+	paths := m.Paths
+	m.Unlock()
+	for name, path := range paths {
```

Comment:
hmm the race is between `GetStats` and `RemovePaths` right ? I think we should also lock RemovePaths in that case.

Also should we lock for the entire duration of GetStats() ? With this lock a cgroup can still be removed in the middle of getting stats out and it can be an error in that case.